### PR TITLE
Update RoleNORManager.md

### PR DESCRIPTION
**Pull request by pfabry**
- Adding message template for wrong issue;
 - Details on dashboard update process.

### CODE REVIEW

File: docs/roles/RoleNORManager.md [hunk 1]
```diff
@@ -6,17 +6,24 @@ title: New Ontology Request (NOR) Manager
 
 The New Ontology Request (NOR) Manager has these basic duties:
 
-1. Receiving `new ontology` requests on OBO and acknowledging the receipt
-2. Including the ontology in the OBO New Ontology Request (NOR) Dashboard
+1. Receiving `new ontology` requests on the OBOFoundry GitHub issues tracker and acknowledging the receipt
```
Comment (matentzn) [inline docs/roles/RoleNORManager.md:9]:
```suggestion
1. Receiving `new ontology` requests on the OBO Foundry GitHub issues tracker and acknowledging the receipt
```

**Review by matentzn — APPROVED** [review]
Small typo but otherwise excellent

File: docs/roles/RoleNORManager.md [hunk 2]
```diff
@@ -6,17 +6,24 @@ title: New Ontology Request (NOR) Manager
...
-- When a new ontology request is received, and the wrong issue template was used, the New Ontology Request (NOR) Manager must inform the NOR requestor to use the correct issue template and close the wrongly submitted issue.
+- The NOR Manager MUST actively listen to issues on the [OBOFoundry GitHub issues tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
+- When a new ontology request is received, and the wrong issue template was used, the NOR Manager must inform the NOR requestor to use the correct issue template and close the wrongly submitted issue with the following message:
+    ```markdown
+    Dear @GITHUBNAME,
+    
+    Thank you for your submission. 
+    However, the wrong issue template has been used. Please make sure to follow the [New Ontology Submission Instructions](https://obofoundry.org/docs/NewOntologyRegistrationInstructions.html) and use the [New Ontology Request issue template](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/new?assignees=&labels=new+ontology&template=new-ontology-request.md&title=).
```
Comment (erik-whiting) [inline docs/roles/RoleNORManager.md:24]: This link goes to a blank issue, not a template (see screenshot). Is that expected?
image

Comment (matentzn) [reply docs/roles/RoleNORManager.md:24]: We should use this lin:

```
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/new?assignees=&labels=new+ontology&template=new-ontology.yml&title=Request+for+new+ontology+%5BNAME%5D
```

**Comment by erik-whiting** [comment]
the link is correct now

File: docs/roles/RoleNORManager.md (hunk 2 above)
Comment (pfabry) [reply docs/roles/RoleNORManager.md]: Document updated with the new link

**Comment by rays22** [comment]
> Looks like this just needs @rays22's review to be ready for merging.

I think you could merge this PR with just two reviews/approvals out of three review requests, but I am adding another approval just in case.

**Review by rays22 — APPROVED** [review]

**Pull request merged**